src/test/cli-integration/rbd/snap-diff.t

```diff
@@ -0,0 +1,47 @@
+  $ ceph osd pool create xrbddiff1 8
+  pool 'xrbddiff1' created
+  $ rbd create --thick-provision --size 1M xrbddiff1/xtestdiff1 --no-progress
+  $ rbd diff xrbddiff1/xtestdiff1 --format json
+  [{"offset":0,"length":1048576,"exists":"true"}]
+  $ rbd rm xrbddiff1/xtestdiff1 --no-progress
+  $ rbd create --size 1M xrbddiff1/xtestdiff1
+  $ rbd diff xrbddiff1/xtestdiff1 --format json
+  []
+  $ rbd snap create xrbddiff1/xtestdiff1 --snap=allzeroes
+  $ rbd diff xrbddiff1/xtestdiff1 --format json
+  []
+  $ rbd diff --from-snap=allzeroes xrbddiff1/xtestdiff1 --format json
+  []
+  $ rbd bench --io-type write --io-size 1M --io-total 1M xrbddiff1/xtestdiff1 > /dev/null 2>&1
+  $ rbd diff xrbddiff1/xtestdiff1 --format json
+  [{"offset":0,"length":1048576,"exists":"true"}]
+  $ rbd diff --from-snap=allzeroes xrbddiff1/xtestdiff1 --format json
+  [{"offset":0,"length":1048576,"exists":"true"}]
+  $ rbd snap create xrbddiff1/xtestdiff1 --snap=snap1
+  $ rbd snap list xrbddiff1/xtestdiff1 --format json | python3 -mjson.tool | sed 's/,$/, /'
```

Comment:
Is there a special reason to use `python3` and not `python`?

Reply:
Python 2 is essentially deprecated and will be unsupported after the end of this year?

Reply:
A propos: https://python3statement.org/ (don't see Ceph on there yet, though)

Reply:
But I believe `python` link will still be available (pointing to `python3`) after python2 is dead?

Actually, I don't have a strong opinion. I just see that in other scripts we use `python` and would like it to be consistent. And when I see `pythonX` it is alarming to me that it might be version dependent.

Reply:
@trociny I, too, would like it to be consistent - ideally by changing all instances of `python -m json.tool` to python3. 

I think at some point far in the future there will be a symlink python -> python3. But for the foreseeable future, at least in RHEL and SUSE, that will not be the case. See https://developers.redhat.com/blog/2018/11/14/python-in-rhel-8/